english/memories/memories.md

```diff
@@ -0,0 +1,7 @@
+What are memories?
+
+Where do they come from?
+
+Why can I only remember the past and not the future?
+
+The first thing I remember is the cold tile floor. The last thing I remember is the dark moment as my eyelids fell as sleep came over me. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.